torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -137,7 +137,7 @@ c10::optional<bool> isDefined(Value* tensor) {
     return true;
   }
   if (tensor->node()->mustBeNone() ||
-      tensor->node()->kind() == prim::Undefined) {
+      tensor->node()->kind() == prim::AutogradZero) {
```

Comment:
This doesn't appear to be reachable. because the isSubtypeOf check above will match any AutogradZeroType node.